broadcast/00-CircuitBreaker.sol/62320/run-1673898399.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Are these the output of runs you've done when testing the branch? 
If so I would say we can remove them from this PR and we'll only keep the ones we actually care about as an outcome of https://github.com/mento-protocol/mento-core/issues/98

Reply:
These runs are actually the ones we care about. I've also created a proposal for them on Baklava